supabase/migrations/43_payment_emails.sql

```diff
@@ -0,0 +1,301 @@
+begin;
```

Comment:
I know you're busy elsewhere, but dropping this feedback for when you're ready. A few things:

* Per our conversation, `paid_tenants` should be inverted to `missing_payment_method`.

* I'd like all of these granular alert views to produce the same table shape: (catalog_name, alert_type, JSON arguments). Then, `alert_all_firing` is the simplest possible UNION ALL of the constituent sub-views.

* Similarly, IIRC we were going for a convention of naming these views `internal.alert_foobar`, where `foobar` is the alert type. This lets views be naturally ordered and grouped in DB resource listings and makes it more obvious what they are.

* Rather than having the edge function collect _all_ alerts whenever it runs, instead have the trigger that _invokes_ the edge function send along the arguments / tokens of a single alert to evaluate. With this design, I don't believe it needs to call back to the DB at all. I'm aware that's not how the current edge function is factored. I've been ... shouting this from the rooftop for some time now.

* nit: rather than `data_not_processed_in_interval_v2` can we call it `data_delay` or something similar ? (Note all of our other discussed alert types are present-tense.)